requirements.txt

```diff
@@ -9,3 +9,8 @@ tqdm
 filelock
 typer
 sh
+re
```

Comment:
I got the following error message when trying to install the packages from this file:

```
ERROR: Could not find a version that satisfies the requirement re (from versions: none)
ERROR: No matching distribution found for re
```

Reply:
Did you mean `pip install regex`?

Reply:
So add `regex` to the requirements file but `import re` in the notebook / py file?